asu/routers/api.py

```diff
@@ -20,9 +20,21 @@ class BuildRequest(BaseModel):
     packages: Union[list, None] = []
     profile: str
     packages_versions: dict = {}
-    defaults: Union[str, None] = None
+    defaults: Optional[
+        Annotated[
+            str,
+            Field(default=None, max_length=20480, description="asu-defaults script"),
+        ]
+    ] = None
     client: Union[str, None] = "unknown/0"
-    rootfs_size_mb: Union[int, None] = None
+    rootfs_size_mb: Optional[
+        Annotated[
+            int,
+            Field(
+                default=None, gt=0, le=1024, description="Resize the root file system."
```

Comment:
Could you add this to the config.py please to make it configurable?

Reply:
All constants removed and now come from `settings...`, including the description for `defaults`.